protocol/communities/manager.go

```diff
@@ -191,6 +192,11 @@ type managerOptions struct {
 	walletConfig           *params.WalletConfig
 	communityTokensService communitytokens.ServiceInterface
 	permissionChecker      PermissionChecker
+
+	// allowForcingCommunityMembersReevaluation indicates whether we should allow forcing community members reevaluation.
+	// This will allow using `force` argument in ScheduleMembersReevaluation.
+	// Should only be used in tests.
+	allowForcingCommunityMembersReevaluation bool
```

Comment:
Is it only to prevent `ForceMembersReevaluation` being called from the clients? Seems a bit too defensive to me to be honest :sweat_smile:

Reply:
Well I tried to be careful! 🧐